Sail/src/Sail/entities/systems/input/GameInputSystem.h

```diff
@@ -1,15 +1,47 @@
 #pragma once
-#include "..//BaseComponentSystem.h"
+#include "../BaseComponentSystem.h"
+#include "Sail/entities/Entity.h"
+class Camera;
+class CameraController;
+
 
 class GameInputSystem final : public BaseComponentSystem {
 public:
 	GameInputSystem();
 	~GameInputSystem();
 
-	void update(float dt) override;
-	
+	// This function is only here because it has to. use update with alpha.
+	void update(float dt) {}
```

Comment:
Add a call to the working update function. Set alpha to 1.0 for no interpolation.